narayana/ArjunaJTA/jta/etc/jbossts-properties.xml

```diff
@@ -9,8 +9,8 @@
         <entry key="HornetqJournalEnvironmentBean.asyncIO">true</entry>
         <entry key="HornetqJournalEnvironmentBean.storeDir">target/narayana</entry>
         <!-- default for NIO is 300 and for AIO it's 2000 -->
-<!--         <entry key="HornetqJournalEnvironmentBean.bufferFlushesPerSecond">7812</entry> -->
-        <entry key="HornetqJournalEnvironmentBean.bufferFlushesPerSecond">300</entry>
+        <entry key="HornetqJournalEnvironmentBean.bufferFlushesPerSecond">20000</entry>
```

Comment:
This setting is considered to be the best in our test environment from heuristic tests (where the transactions per second per thread are around '16847' , see  https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/job/narayana-performance/37/artifact/bm-output.txt). 
Different environments might need different configurations in order to perform at their best.